### ISSUE TITLE
Sometimes mailCow getting stuck

### ISSUE THREAD

**Issue by mihailsalari** [mailcow/mailcow-dockerized#5731]
I use hetzner, and, for more than 2 years mailcow. At the beginning, it was working pretty fine, then the space was increasing drastically, and I reistalled mailcow. Then, the syncing process between apple mail and server just dissapear. Now, the mailcow server stuck, I don't know why and planned to move to another open source solution in case won't find and solve the problem

**Comment by bundyland** [comment]
Have you disabled SOLR and CLAMD? Otherwise 4 gig of RAM aren’t enough for Mailcow. That could be the reason why your cow hangs.

Hetzner offers also ARM cloud server with 8 gig of RAM for about 7€ a month. This is what I have and the cow runs absolutely smoothly.

**Comment by mihailsalari** [comment]
> Have you disabled SOLR and CLAMD? Otherwise 4 gig of RAM aren’t enough for Mailcow. That could be the reason why your cow hangs.
> 
> Hetzner offers also ARM cloud server with 8 gig of RAM for about 7€ a month. This is what I have and the cow runs absolutely smoothly.

No, I haven't enabled these. Only UWF and fail2ban

**Comment by mihailsalari** [comment]
<img width="1521" alt="Screenshot 2024-02-10 at 12 07 19 AM" src="https://github.com/mailcow/mailcow-dockerized/assets/23311250/adb81832-f6f0-4a10-aa3e-d83641517139">


> Have you disabled SOLR and CLAMD? Otherwise 4 gig of RAM aren’t enough for Mailcow. That could be the reason why your cow hangs.
> 
> Hetzner offers also ARM cloud server with 8 gig of RAM for about 7€ a month. This is what I have and the cow runs absolutely smoothly.

Are you talking about this?

**Comment by bundyland** [comment]
Yes, this is the server I have, but you need to add around 1€ a month for the IPv4.

Just to be clear: you need to disable SOLR and CLAMD in the mailcow.conf file. After that docker compose down && docker compose up -d. That should work as well with a 4 gig machine. All these is also mentioned in the documentation of Mailcow.

BTW: any kind of firewall is not recommended if you use Mailcow. This can make some trouble.

**Comment by mihailsalari** [comment]
> you need to disable SOLR and CLAMD

Thanks a lot.

I actually used this guide to install mailcow:
https://gist.github.com/opentaq/2ce85108bee528700477c6db89610968

Also, I use on hetzner these settings for the firewall:
<img width="1055" alt="Screenshot 2024-02-10 at 12 17 07 AM" src="https://github.com/mailcow/mailcow-dockerized/assets/23311250/58c44994-d237-432f-881a-410b1b6850ca">

**Comment by mihailsalari** [comment]
<img width="919" alt="Screenshot 2024-02-10 at 12 29 33 AM" src="https://github.com/mailcow/mailcow-dockerized/assets/23311250/3fa00ee2-a871-4842-b9db-e1612f6ff45a">
just followed your advice, then run docker stop then up, and here's the updates. It shows 19/20, instead of 20/20. What are your thoughts on this?

**Comment by bundyland** [comment]
I don't know exactly, but I guess this is normal, because you have disabled (hopefully) SOLR and CLAMD.

If you enter `docker ps` all containers should be "up" and some of them with status "healthy". If this is not the case, you should try a reboot.

BTW: you should only follow the official documentation of Mailcow here:
[https://docs.mailcow.email/](url)

**Comment by ShreshthTiwari** [comment]
`docker ps` is showing all containers as Up and 2 as healthy still after some time the web interface is freezing and is inaccessible.
It happening only after i updated to the current latest version.
CPU usage is 5% and RAM usage is around 60% in the web interface before freeze so i don't think its the memory overflow issue.

<img width="1921" height="1080" alt="Image" src="https://github.com/user-attachments/assets/49898a8b-afe8-4ff3-a398-6ac13c6844da" />

<img width="949" height="223" alt="Image" src="https://github.com/user-attachments/assets/1e9df336-1510-47ee-8a31-a98d0c304615" />

**Comment by patschi** [comment]
This issue is more than one year old and might have been a completely different reason. Please don't mix up issues. If you have more detail and it is reproducible across systems, feel free to create a new issue. 

Note that GitHub issues is not for support queries.
